webapp/src/components/Modals/BuyWithCryptoModal/MintNameWithCryptoModal/MintNameWithCryptoModal.tsx

```diff
@@ -103,7 +104,12 @@ const MintNameWithCryptoModalHOC = (props: Props) => {
   const onGoBack = useCallback(() => {
     onClose()
     onOpenFatFingerModal()
-  }, [])
+  }, [onOpenFatFingerModal, onClose])
+
+  const onCloseModal = useCallback(() => {
+    onCloseFatFingerModal()
+    return onClose()
```

Comment:
```suggestion
    onClose()
```
just to keep it consistent with the one above

Reply:
As we're replacing the `onClose` method from the original one to this one, the `BuyWithCryptoModal` expects the type of the method to return the same type as the original `onClose`. I could either keep the return there or change the type in the `BuyWithCryptoModal` what do you find best?